config/application.properties

```diff
@@ -0,0 +1,6 @@
+
+application.info=demo sources for work with files presentation
+java.version=1.8
+
+excel.defaultName=excelFileFromLocationA
```

Comment:
Usually we don't want to keep something like excelFromA/B. Just proper working defaults.